src/main/remote-sync/helpers.ts

```diff
@@ -0,0 +1,85 @@
+import Store from 'electron-store';
+
+export const remoteSyncStore = new Store<{
+  lastFilesSyncAt?: string;
+  lastFoldersSyncAt?: string;
+}>();
+
+export const clearRemoteSyncStore = () => remoteSyncStore.clear();
+export function getLastFilesSyncAt(): Date | undefined {
+  const value = remoteSyncStore.get('lastFilesSyncAt');
+
+  if (!value) return undefined;
+
+  return new Date(value);
+}
+
+export function saveLastFilesSyncAt(date: Date, offsetMs: number): Date {
+  remoteSyncStore.set(
+    'lastFilesSyncAt',
+    new Date(date.getTime() - offsetMs).toISOString()
+  );
+  return date;
+}
+
+export function getLastFoldersSyncAt(): Date | undefined {
+  const value = remoteSyncStore.get('lastFoldersSyncAt');
+
+  if (!value) return undefined;
```

Comment:
May we return the new Date with the beggining of the epoch to isolate here all the logic of returning a Date depending on if exists or not?

Reply:
I think we should not here, since this basically gives you a date, or doesn't give you a date (which means no sync has ever happened), if we are going to return always a date, we won't be able to determine if there was a previous sync at some Date